src/_data/site.js

```diff
@@ -16,7 +16,10 @@ const twitterProfile =
   twitterURL !== 'twitter' ? getTwitterProfile(twitterURL) : '@freecodecamp';
 
 module.exports = async () => {
-  const site = await sourceAPI.settings
+  // To do: Combine sensible defaults for all locales here and make sure any
```

Comment:
```suggestion
  // TODO: Combine sensible defaults for all locales here and make sure any
```

Will show up as an actual todo on IDEs

Reply:
Awesome, thanks for catching that. I had no idea that IDEs have that feature. I'll keep this in mind going forward.